### ISSUE TITLE
Accessibility Issue: Violation of WCAG 2 AA contrast ratio thresholds /2019/11/19/travailler-ouvertement-partie-1.html

### ISSUE THREAD

**Issue by Moro-Code** [sara-sabr/ITStrategy#720]
**Description**
Elements have insufficient color contrast of 3.81 (foreground color: #e83e8c, background color: #ffffff, font size: 10.5pt (14px), font weight: normal). Expected contrast ratio of 4.5:1

relates to #714 

**Device Information**  
*Device*:  
*Operating System (incl. version)*:  
*Browser (incl. version)*:  
*Additional Details*:

**Element(s) Information**  
*Page*:  https://sara-sabr.github.io/ITStrategy/2019/11/19/travailler-ouvertement-partie-1.html
*CSS Selector(s) (if possible and applicable)*:  
```code:nth-child(1)```
```code:nth-child(2)```
*Additional Identifier(s) (if possible and applicable)*:  
*Additional Details*:  

**Screenshot(s) (if possible)**

<img width="739" alt="Screen Shot 2020-01-30 at 3 27 36 PM" src="https://user-images.githubusercontent.com/26819992/73487483-19204000-4375-11ea-8d78-1789e3d1d642.png">

**Additional information or thoughts**

As a result of scan #659
